portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,41 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+//    public ArrayList<String> comments = new ArrayList<String>(List.of("boom", "bam", "pow"));
+
+
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // obtain input from form(the user's comment)
+    String userComment = getParameter(request, "userCommentInput", "");
+    
+    // seperate words into indivdual text
```

Comment:
This comment doesn't add much value. The line of code pretty much speaks for itself.